calc/src/mechanics/gen78.ts

```diff
@@ -1008,6 +1008,11 @@ export function calculateSMSS(
     desc.isProtected = true;
   }
 
+  if (field.defenderSide.isProtected && attacker.isDynamaxed) {
```

Comment:
I think we should actually be able to combine this conditional with the `if` on line 1006? Something like

```ts
if (field.defenderSide.isProtected &&
	((move.isZ && attacker.item && attacker.item.includes(' Z')) ||
	  attacker.isDynamaxed)) {
```

Reply:
We can do that, i just thought it would be more readable this way?

Reply:
Considering the lines are doing the same thing, I don’t think readability is a concern. We also have much longer if statements than this so we should be good :)

Reply:
Alright, just applied the change.